### DISCUSSION TITLE
Why not implementing the legacy Subsonic API?

### DISCUSSION THREAD

**Discussion by poisonborz** [jeffvli/feishin#253] [Q&A]
There would be a lot of reasons to add it - there are great alternative implementations on server side too, like Gonic.

**Comment by jeffvli** [comment]
I'm planning on adding it soon. I was initially planning on waiting until [OpenSubsonic](https://github.com/opensubsonic/open-subsonic-api) began implementation but it has taken longer than expected. Subsonic's API is currently missing some important pagination features which I use with Navidrome/Jellyfin but I've thought of a way to work around it for now.

**Reply by dahlbergc** [reply]
There was an update recently to the [Gonic](https://github.com/sentriz/gonic/releases/tag/v0.16.0) project.

Gonic seems to be the most active project using the Subsonic API that follows OpenSonic, but I'm unsure if this release includes the pagination features you're looking for.

**Comment by ghost** [comment]
hello. is there any progress on this? would love to use feishin with nextcloud music.. thanx!

**Comment by offobject** [comment]
Any updates on Subsonic? There are currently not really any great clients for Astiga aside from Sonixd, and that seems to be slowly getting buggier... would be super super appreciated.

**Reply by jeffvli** [reply]
Subsonic support was added in 0.10.0.

**Reply by offobject** [reply]
Woah thank you! I'd been waiting around for the readme to change.

**Comment by ghost** [comment]
thank you!!

**Comment by poisonborz** [comment]
Awesome news! The readme should be updated

![image](https://github.com/user-attachments/assets/8f3d55bf-2415-48e9-83f3-051005bf6b44)
